modules/accounts/email.nix

```diff
@@ -81,6 +81,14 @@ let
           Note, if set then the system default may still be accepted.
         '';
       };
+
+      fingerprint = mkOption {
```

Comment:
Since this is a new option in the email infrastructure it will affect all related modules. If possible, they should be given support for the option, otherwise they should warn or error out if the option is set. You'll need to do some research on the different supported tools to see what is possible.

Reply:
For now i moved this option to msmtp-accounts due to:
- most other tools don't support this option
- the ones who do expect another format ( e.g. offlineimap - which only supports fingerprints on non STARTTLS connections )

However i'm not sure if this was the right decision, while this keeps the global interface clean it separates the fingerprint from the rest of the tls options.

What do you think about this?

Reply:
Moving it to the module specific place is the correct solution. The global options should be reserved for things that can be supported by a majority of modules.